### ISSUE TITLE
"Statistics" option of scatterplot and graph of self managed display is giving an error

### ISSUE THREAD

**Issue by Coryv221** [PardeeCenterDU/IFs-Issues-Tracking#6]
Error short description/title: Statistics of scatterplot and graph of self managed display is giving an error
Date found: 11/23/2020
IFs Version (number, VB6/.NET, Install/Development): IFs 7.56 .NET install
Person found by: Suraj Thapa
Type of error (see below): Display issues
Description of error: The option called “Statistics” under Graph. It gives an error called “Error Handler for IFsWeb.” (The link says "Analysis (Statistics) computes standard regression statistics for any kind of line chosen (linear by default)")
Steps to replicate the error, if applicable: Using Pardee wiki - link – do as it says about scatter plots for self-managed display –

Go to Self-Managed display. 2. Select country/regions, any variable (I chose GDPPCP and TFR), and just base working model, then go to display 3. Choose all country and go to display 4. Go to Graph Tab 5. Go to "Scatter Plot" 6. Go to Analysis 7. Go to "Statistics" 8.Sometimes you will get an error (mostly, if you have not used a flexible display after opening IFs ) and sometimes it works fine (if you have used flex display)
Attempt to recreate error in VB6 or .NET depending on where found and note whether error reproducible: It was on windows and .NET
[statistics_under_scatterplot.docx](https://github.com/PardeeCenterDU/IFs-Issues-Tracking/files/6093546/statistics_under_scatterplot.docx)

**Comment by PardeeCenterIFs** [comment]
JRS (Jan 14, 2021): Difficult one, can reproduce in installation machine, but not in my development machine, need to keep testing.

Suraj (Feb 2, 2021): Thank you. We will keep checking with new versions. I just tried again on 7.61 (vb6) and it was fine.
On 7.62 .NET, I am still getting an error.

**Comment by PardeeCenterIFs** [comment]
JRS: I have now learned that if I replace the entire ifsWeb folder with a non-compiled version, then it works fine.
I also learned that in my development machine the compiled code works fine.
Finally I learned that the clr.dll in my machine and the target machines is not identical, mine seems to be newer (ver 4.8.4341.0), this file seems to be the one producing the error in the target machines, I found it using event viewer.
